app/controllers/shows_controller.rb

```diff
@@ -0,0 +1,7 @@
+require 'csv'
+require 'pry-byebug'
+class ShowsController < ApplicationController
+    def index 
+    	@shows = CSV.parse(File.read('lib/netflix_titles.csv'), headers: true).first(10)
```

Comment:
there are a couple of magic variables here - perhaps `10` could move to a constant called `NUMBER_OF_SHOWS` and your file path could move to a variable called `show_titles_file`?

it would then look like this:
```
@shows = CSV.parse(File.read(show_titles_file), headers:true).first(NUMBER_OF_SHOWS)
```
WDYT?

Reply:
Agree and done